docs/source/models/concept_models/vivarium_mncnh_portfolio/facility_model.rst

```diff
@@ -0,0 +1,97 @@
+:orphan:
+
+.. _2024_facility_model_vivarium_mncnh_portfolio:
+
+Delivery Facility Choice Model
+==============================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Background
+----------
+
+To capture the complex relationship between choice of delivery facility (home birth vs a facility with basic emergency obstetric and neonatal care [BEmONC] vs a facility comprehensive care [CEmONC]), the belief about gestational age (believed pre-term vs believed full term), and the related factors of antenatal care (ANC), and low birth weight small for gestational age (LBWSG) risk exposure, we will include two novel affordances in our simulation: (1) correlated propensities for ANC, home delivery, and LBWSG; and (2) conditional probabilities for home delivery that differ based on the believe pre-term/full-term status when labor begins.
+
+Coming up with values for these correlations and conditional probabilities that are consistent with GBD and external evidence is detailed at the end of this document.  But before we get to that complexity, let's start with how we will use these correlations and conditional probabilities in the simulation.
+
+[[Causal model diagram goes here]]
+
+Correlated propensities
+-----------------------
+
+This section describes how we will model an "instrisic correlation" of ANC, home delivery, and LBWSG.  In short, we will use a Gaussian copula to model this, which has three parameters capturing the correlation between each pair of the three propensities.
+
+The motivation for these correlations is as follows: we hypothesize that there are important "common causes" that are not shown explicitly in the diagram above.  For example, having a home delivery and having no ANC visits might both be influenced by rurality --- if all health services are offered far away, it is logical that people will be able to access them less.
```

Comment:
```suggestion
The motivation for these correlations is as follows: We hypothesize that there are important "common causes" that are not shown explicitly in the diagram above.  For example, having a home delivery and having no ANC visits might both be influenced by rurality --- if all health services are offered far away, it is logical that people will be able to access them less.
```

Reply:
Since it is after a colon, I perfer to keep it lowercase.

Reply:
[when to capitalize after a colon](https://www.google.com/search?q=when+to+capitalize+after+a+colon&rlz=1C5GCEM_enUS1023US1023&oq=when+to+capitalize+after+a+colon&gs_lcrp=EgZjaHJvbWUqDggAEEUYJxg7GIAEGIoFMg4IABBFGCcYOxiABBiKBTIICAEQABgWGB4yCAgCEAAYFhgeMggIAxAAGBYYHjIKCAQQABgKGBYYHjIICAUQABgWGB4yCAgGEAAYFhgeMggIBxAAGBYYHjIICAgQABgWGB4yCAgJEAAYFhge0gEIMTQwMmowajeoAgCwAgA&sourceid=chrome&ie=UTF-8)